source/_patterns/02-organisms/components/content-aside.mustache

```diff
@@ -30,4 +30,6 @@
     </div>
   {{/timeline}}
 
+  <div class="divider"></div>
```

Comment:
is this divider needed even if timeline is optional?

Reply:
Yes, we need that divider on small screens too, in this case we can separate the aside from the content:

![image](https://user-images.githubusercontent.com/111341278/212906453-2ad96593-e7a8-4912-985a-d559250b4c2d.png)